src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -183,7 +183,7 @@ public final class Opt {
       Log.error("--nodedup is not supported with --ignore or --keep_going");
       exit(1);
     }
-    if (instrumentOnly && dumpClassesDir.isEmpty()) {
+    if (instrumentOnly != null && !instrumentOnly.isEmpty() && dumpClassesDir.isEmpty()) {
```

Comment:
As with the checks of the other parameters above, we can rely on `instrumentOnly` to never be `null`.